<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountWindow</class>
 <widget class="QMainWindow" name="AccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>BlackList</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Contacts</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QListView" name="blackList"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Search for new contact</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QListView" name="listView_2"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="accountNameLabel">
        <property name="text">
         <string>Account:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccounts">
    <property name="title">
     <string>Accounts</string>
    </property>
    <addaction name="actionExit_to_main_menu"/>
    <addaction name="actionChange_account_name"/>
    <addaction name="actionChange_account_password"/>
   </widget>
   <addaction name="menuAccounts"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit_to_main_menu">
   <property name="text">
    <string>Exit to main menu</string>
   </property>
  </action>
  <action name="actionChange_account_name">
   <property name="text">
    <string>Change account name</string>
   </property>
  </action>
  <action name="actionChange_account_password">
   <property name="text">
    <string>Change account password</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
